integration-tests/plugins/index.js

```diff
@@ -292,10 +292,15 @@ module.exports = on => {
         elite2api.stubScheduledEventsForThisWeek(thisWeeksSchedule),
         elite2api.stubScheduledEventsForNextWeek(nextWeeksSchedule),
       ]),
-    stubAppointmentsGet: appointments => Promise.all([elite2api.stubAppointmentsGet(appointments)]),
```

Comment:
not sure why these were `Promise.all` since only one promise?

Reply:
Agreed.